doc/external-libs/ginkgo.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
+	  "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+  <head>
+    <title>The deal.II Readme on interfacing to Ginkgo</title>
+    <link href="../screen.css" rel="StyleSheet">
+    <meta name="copyright" content="Copyright (C) 2008 - 2018 by the deal.II authors">
+    <meta name="keywords" content="deal.II">
+  </head>
+
+  <body>
+
+    <h1>Interfacing <acronym>deal.II</acronym> to Ginkgo</h1>
+
+    <p>
+        <a href="https://github.com/ginkgo-project/ginkgo"
+           target="_top">Ginkgo</a> is a
+        Numerical linear algebra software package that provides its users with
+        highly optimized fine grid level linear algebra operations. It currently has
+        the capability to solve on the CPU with the support of OpenMP and on the GPU
+        with NVIDIA's cuda libraries. For some operations it uses NVIDIA's own libraries, CuSparse
+        and CuBLAS but some routines also have self implemented cuda functions. See the documentation
```

Comment:
It would be good to clarify if the phrase "the documentation" refers to Ginko's official documentation, or that of the wrappers within the deal.II library.

Also, can you hyperlink the phrase "the documentation" to relevant documentation?